bundles/org.openhab.binding.modbus/src/main/feature/feature.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <features name="org.openhab.binding.modbus-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>file:${basedirRoot}/bundles/org.openhab.io.transport.modbus/target/feature/feature.xml</repository>
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
```

Comment:
IIRC this is not working, because the transport bundle is not available (it‘s part of addons). Therefore the feature verification will fail.

Reply:
Do we have any documentation on what how to set repository element? This is a bit confusing to me...

Reply:
Agreed. In 99+% of the bundles what you did is correct. The problem arises from the fact that the bundle depends on a feature that is not part of core but of addons. The question is, if we should integrate the transport bundle in the aggregated feature, too. Then we could add all transport bundles (including dependencies) in this feature.

Reply:
Ok, thanks, I have reverted the repo entry.

Reply:
> The problem arises from the fact that the bundle depends on a feature that is not part of core but of addons

I was just browsing the addons repo when contemplating about https://github.com/openhab/openhab-core/issues/1319#issuecomment-616226058 - and I came to the conclusion that the io.transport.modbus bundle simply does not belong in this repo, but should be moved to openhab-core, where all other io.transport bundles reside.
We can't move it for 2.5, but I'd strongly suggest that we move it for 3.0.